src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
 
 Once executed, this query will return a single Workflow named _Create OpenStack Instance_.
 
+=== Send Workflows to another Catalog Bucket, from another Scheduler or the local one.
+
+In the Workflow Catalog, it is possible to select Workflows from the selected bucket and send them to another bucket. This bucket can belong to the same Catalog or to another Scheduler's Catalog.
+
+==== Send Workflows directly
+
+If the Workflows to share are sent to a Scheduler having a public API, or to another bucket of the local Scheduler, it is convenient to send them directly.
```

Comment:
[...] are sent to a bucket within the same local Scheduler or to any Scheduler having a public API, it is [...]